packages/components/src/components/form/Radio/Radio.tsx

```diff
@@ -64,6 +64,10 @@ const RadioIcon = styled(CheckContainer)`
         }
     }
 
+    input:disabled + && {
```

Comment:
Also, can't we use just `input:disabled + &&`?

Reply:
Hmm, I am not sure I understand..

Reply:
Sorry, I mean try to use `input:disabled + &`